scripts/build_mod_ic.py

```diff
@@ -990,6 +990,28 @@
         },
     },
 
+    # VQFN-32 from BQ40Z60
+    # IPC-7351B: QFN26P500X500X100-33N
+    "QFN-32-EP-TI": {
+        "rows": 4,
+        "pins": 32,
+        "pin_pitch": 0.5,
+        "row_pitch": 4.7,
+        "pad_shape": (0.6, 0.25),
+        "ep_shape": (3.45, 3.45),
+        "ep_paste_shape": (1.0, 1.0, 1.5, 1.5),
+        "ep_mask_shape": (2.0, 2.0, 0.5, 0.5),
+        "ep_vias": (0.4, 0.6, 1.9),
```

Comment:
Probably want to use 0.3mm via drills here, closer to the recommended 0.2 and still within most specs.